src/cmd/flux-resource.py

```diff
@@ -446,14 +468,15 @@ def resources_uniq_lines(resources, states, formatter):
 
     #  Create a mapping of resources sets that generate uniq "lines":
     lines = {}
+    config = flux.Flux().rpc("config.get").get()
```

Comment:
This seems like a reasonable approach to me! Possibly this is just prototype code at this point, and if so ignore my comment, but the fetch of the config should probably go outside of this function into `list_handler()` directly, at least so that only a single flux handle is required.

On another note, it is a bit unfortunate that queues and their properties will now be duplicated in output -- an unfortunate side effect of the design that resources assigned to a queue are selected by properties. I guess we could try removing properties that are the same as the queue name, then the PROPERTIES column would go away when they match exactly those in QUEUE,  but this seems like it could end up being more confusing...

Reply:
> On another note, it is a bit unfortunate that queues and their properties will now be duplicated in output -- an unfortunate side effect of the design that resources assigned to a queue are selected by properties. I guess we could try removing properties that are the same as the queue name, then the PROPERTIES column would go away when they match exactly those in QUEUE, but this seems like it could end up being more confusing...

yeah, @garlick mentioned that in #4765.  There's a part of me that wants to leave them there, b/c while we often name the properties the same names as the queues, you don't have to.  I dunno, something we can debate as I flush this out more.

Reply:
What if you only remove the queue property from the properties list if it matches the queue name. I think keeping the output of `flux resource list` as simple as possible should be a goal here, not just to satisfy the exact request.

Then, this could allow us to rethink how queues are configured to make this less confusing, without eventually breaking or changing the output of `flux resource list`.

Reply:
hmmm, i guess this is where it gets tricky.  Should it be removed from PROPERTIES only if the user is going to output QUEUE?  Or should it not matter if the user is outputting QUEUE?

Reply:
Good question. A user or sysadmin may want to be able to query the set of all properties with `flux resource list -no {properties}`, or `flux resource list -no '{properties:<25} {nodelist}'`. Always removing queue properties from PROPERTIES would somewhat thwart this use case. However, I can see why you would lean toward doing this since it might _also_ be confusing if the listed PROPERTIES change based on what headings are used. However, if we take that approach, then we are effectively making queue properties "hidden". Of course, we could add an option to `flux resource` to change this behavior, and then pick one as the default.

Reply:
perhaps we could have an alternate `properties` key?  like `properties_simple` or something.

Reply:
That is a pretty good idea! :+1: Maybe `properties_redacted` or `properties_other` or just `propertiesx` (for properties-excluded). Anything is probably fine since this would likely be in the default format, but not used too often when specifying a format manually.